### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jeriox**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (felixrindt) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: `Ex`?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (felixrindt) [inline .github/ISSUE_TEMPLATE/bug-report.md:35]: Can we shorten the hardware section? I think for most bug reports this info is not relevant, but it always clutters those templates IMO.

Comment (felixrindt) [reply .github/ISSUE_TEMPLATE/bug-report.md:35]: Propose:

> **Environment**
> State which device, operating system, browser and browser version you are using.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (jeriox) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: i think this is short for example ^^

Comment (felixrindt) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: hmm. put it in parens?

**Review by felixrindt — APPROVED** [review]

**Pull request merged**